### PR TITLE
Admin create tour

### PR DESCRIPTION
**Pull request by conh-2027**
>>> Title: 
Admin create tour
>>> Redmine ticket: 
https://edu-redmine.sun-asterisk.vn/issues/17685
>>> Attachment: 
Screenshot from 2019-09-20 19-12-56
Screenshot from 2019-09-23 18-04-57
Screenshot from 2019-09-23 18-04-33

### CODE REVIEW

**Comment by conh-2027** [comment]
ready

File: app/views/admin/tours/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<% provide :title, t(".new") %>
+<%= provide :button_text, t(".create") %>
+<div class="row justify-content-center">
+  <div class="col-9">
+    <%= render 'form' %>
```
Comment (hungct-1121) [inline app/views/admin/tours/new.html.erb]: nháy đôi nak

File: app/views/shared/_flash.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<% flash.each do |msg_type, msg| %>
+  <%= content_tag(:div, msg, class: "alert alert-#{msg_type}") %>
```
Comment (hungct-1121) [inline app/views/shared/_flash.html.erb]: bỏ ()

**Comment by conh-2027** [comment]
ready

**Comment by hungct-1121** [comment]
(ok) merged

**Pull request merged**